src/DotNetCore.CAP.AzureServiceBus/ITransport.AzureServiceBus.cs

```diff
@@ -33,11 +36,26 @@ internal class AzureServiceBusTransport : ITransport
 
         public BrokerAddress BrokerAddress => new BrokerAddress("AzureServiceBus", _asbOptions.Value.ConnectionString);
 
+        /// <summary>
+        /// Gets a custom Producer
```

Comment:
What do you think of placing something like: 

Creates a <see cref="ServiceBusProducerDescriptor" /> for the given message. If there's no custom producer configured to it in the <see cref="AzureServiceBusOptions" />, it will be created one with the default TopicPath configured there.

Reply:
I am now using "Creates a producer descriptor for the given message. If there's no custom producer configuration for the message type, one will be created using defaults configured in the AzureServiceBusOptions (e.g. TopicPath)."

What do you think?

Reply:
Looks great!